scripts/ci/test_plan.py

```diff
@@ -103,6 +103,9 @@ def __init__(self, modified_files, pull_request=False, platforms=[], detailed_te
         self.platforms = platforms
         self.default_run = False
         self.detailed_test_id = detailed_test_id
+        self.ignore_path = f"{zephyr_base}/scripts/ci/twister_ignore.txt"
```

Comment:
maybe set this to the default in add_argument below and here just set self.ignore_path...